bundles/net.tourbook/src/net/tourbook/messages.properties

```diff
@@ -3215,6 +3215,7 @@ Tour_Database_PostUpdate_037_SetHasGeoData        = Data update 37: Set geo data
 Tour_Database_PostUpdate_040_SetTourRecordingTime = Data update 40: Compared tour - {0} of {1}
 Tour_Database_PostUpdate_043_LatLonE6             = Data update 43: Converting lat/lon \u2192 E6 - {0} of {1} - {2} % - {3} \u0394 
 Tour_Database_PostUpdate_047_Weather              = Data update 47: Converting weather data - {0} of {1} - {2} % - {3} \u0394 
+Tour_Database_PostUpdate_048_Weather_Clouds       = Data update 48: Fixing the weather clouds texts - {0} of {1} - {2} % - {3} \u0394 
```

Comment:
One minor issue, the text is wrapped and clipped, does this happen also in your dev environment?

![image](https://user-images.githubusercontent.com/1283445/164403842-1df71ee4-d497-42cc-9cd3-da27525b89a2.png)

It seems that this text is larger than a max text length

![image](https://user-images.githubusercontent.com/1283445/164404030-730f67d6-84ce-4c48-927f-9ec6af05587f.png)

Reply:
Yes this happens also (english and french). I have fixed it for both languages. However, translators should be let known of that possible issue as it could happen in other languages too.

English
![image](https://user-images.githubusercontent.com/25235665/164507891-6cc2298a-c873-48a2-a5dc-a70d1144fc77.png)

French
![image](https://user-images.githubusercontent.com/25235665/164508327-cd892a33-4d5d-4978-a6a5-6b5b8697fcff.png)

Reply:
Better...

![image](https://user-images.githubusercontent.com/1283445/164606557-0ad63281-700d-4eb9-8f25-42714beac0d8.png)

Shouldn't we switch to a common text, e.g.

`Data update %d: %d of %d - %d % - %d Δ`

Reply:
We could but the user would lose the information about what the update does.

Reply:
With your suggestion, it now looks like this with my latest PR update

![image](https://user-images.githubusercontent.com/25235665/164816961-5531a498-5603-4039-bd49-3d85c90a11c7.png)

Reply:
> We could but the user would lose the information about what the update does.

We can put this info into the release notes. The info in the splash screen would also not work when there are multiple changes.

Reply:
> With your suggestion, it now looks like this with my latest PR update
> 
> ![image](https://user-images.githubusercontent.com/25235665/164816961-5531a498-5603-4039-bd49-3d85c90a11c7.png)

The splash screen looks a bit empty when MT has only 2 tours but now there is enough space for 9.999.999... tours

Reply:
With more tours it looks less empty

![image](https://user-images.githubusercontent.com/1283445/164882322-490b60b9-7a69-45b8-b680-6b0ff4a3d897.png)

Reply:
> > We could but the user would lose the information about what the update does.
> 
> We can put this info into the release notes. The info in the splash screen would also not work when there are multiple changes.

True, it's a good idea to put it in the release notes